plugins/x86/x86_opcode.ml

```diff
@@ -2,26 +2,17 @@ open Core_kernel.Std
 open Bap.Std
 open Regular.Std
 
-include X86_opcode_btx
-include X86_opcode_cmps
-include X86_opcode_cmpxchg
-include X86_opcode_ins
-include X86_opcode_lods
-include X86_opcode_mov
-include X86_opcode_movs
-include X86_opcode_outs
-include X86_opcode_scas
-include X86_opcode_stos
+type t = 
```

Comment:
I don't like it, this is just an attempt to move the problem into another dimension, without solving it. 
We either should get rid of opcode type, or implement it as an abstract or extensible type.